server/src/test/java/server/haengdong/application/BillActionServiceTest.java

```diff
@@ -0,0 +1,69 @@
+package server.haengdong.application;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.assertj.core.api.Assertions.tuple;
+
+import java.util.Comparator;
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import server.haengdong.application.request.BillActionAppRequest;
+import server.haengdong.domain.BillAction;
+import server.haengdong.domain.Event;
+import server.haengdong.persistence.BillActionRepository;
+import server.haengdong.persistence.EventRepository;
+
+@SpringBootTest
+class BillActionServiceTest {
+
+    @Autowired
+    private BillActionService billActionService;
+
+    @Autowired
+    private EventRepository eventRepository;
+
+    @Autowired
+    private BillActionRepository billActionRepository;
+
+    @DisplayName("지출 내역을 생성한다.")
```

Comment:
```suggestion
    @DisplayName("지출 내역을 생성한다")
```

a: 사소한 컨벤션인데, 이 부분도 함께 정해 보아요